x-pack/plugin/sql/sql-action/src/main/java/org/elasticsearch/xpack/sql/action/AbstractSqlQueryRequest.java

```diff
@@ -57,6 +63,7 @@
     static final ParseField FILTER = new ParseField("filter");
     static final ParseField MODE = new ParseField("mode");
     static final ParseField CLIENT_ID = new ParseField("client_id");
+    static final ParseField CLIENT_VERSION = new ParseField("client_version");
```

Comment:
I would just say `version` since all parameters are client specific. `client_id` has is as such since just `id` would have been too cryptic.
There's no client_mode, or client_params so why client_verision.

In fact, if anything we could change client_id to drop the prefix by using a different word: something like fingerprint or info (too loose).

Reply:
Makes sense, fixed it.